data/transform.py

```diff
@@ -78,22 +78,28 @@ def __call__(self, sample):
 
 class Normalize(object):
     """" Normalize a tensor image with mean and standard deviation
-    Given mean: ``(M1,...,Mn)`` and std: ``(S1,..,Sn)`` for ``n`` channels, this transform
+    Given mean: ``(M1,...,Mn)`` and std: ``(S1,..,Sn)`` for ``n`` channels, if not None, this transform
     will normalize each channel of the input ``torch.*Tensor`` i.e.
     ``input[channel] = (input[channel] - mean[channel]) / std[channel]``
+    else
+    ``input[channel] = input[channel] / 255
     """
-    def __init__(self, mean, std):
+    def __init__(self, mean=None, std=None):
         self.mean = mean
         self.std = std
 
-    def __call__(self, tensor):
+    def __call__(self, sample):
         """
         Args:
-            tensor (Tensor): Tensor image of size (C, H, W) to be normalized.
+            sample (Tensor): Tensor image of size (C, H, W) to be normalized.
         """
-        for t, m, s in zip(tensor, self.mean, self.std):
-            t.sub_(m).div_(s)
-        return tensor
+        if not self.mean or not self.std:

```

Comment:
`image = sample["image"]
if self.mean is not None or self.std is not None:
    image.div_(255)
else:
    image.sub_(self.mean).div_(self.std)
sample["image"] = image
return sample
`